tools/bazel.rc

```diff
@@ -2,5 +2,8 @@
 build --verbose_failures
 test --test_output=errors
 
+# Release Build
+build:release -c opt
```

Comment:
the 2x benefits I measured is with envoy (c++) but I expect debug to be -gcflags="-N -l" and opt to be without those and thus make a difference, not sure the bazel go rules do do that (we should check, but even if they don't today, having 2 or more config will be useful in the future)